api/locations/locationsRoutes.js

```diff
@@ -0,0 +1,53 @@
+const express = require("express");
+const router = express.Router();
+const Place = require("../../data/helpers/locationHelper");
+
+router.use(express.json());
+
+router.get('/', (req, res) => {
+    Place.getAllPlaces()
+        .then(place => {
+            if(place) {
+                res.status(200).json(place)
+            } else {
+                res.status(404).json("Error: No places are in the database")
+            }
+        }) .catch(e => {
+            res.status(500).json(`Message: ${e}`);
+        })
+});
+
+router.get('/:id', (req, res) => {
+    Place.getPlaceById(req.params.id)
+        .then(place => {
+            if(place) {
+                res.status(200).json(place)
+            } else {
+                res.status(404).json('Error: Could not find that location')
```

Comment:
Looks like a small typo? Is this the structure? `{e: 'e message'}`

Reply:
Wait where? Lol which line

Reply:
Line 26. Sorry that's probably nit picking.

Reply:
oh no, I did that before the catch. To test if there was a "place" if not then I spit out you didn't put in a place, any status 500 errors will show in the catch on line 28